src/object_functions.c

```diff
@@ -1415,6 +1415,9 @@ void play_tt_voice_clip(u16 soundID, s32 interrupt) {
     }
 }
 
+/**
+ * Fish init behaviour.
```

Comment:
I'm not a fan of the empty comments. I would not count this as "documented". I know there are plenty of functions in this repo that don't have great documentation, but we gotta put *something* there for the random person who stumbles across these functions.